### PR TITLE
NFDIV-784: Integration with Send-Letter Service for NFD Bulk Print

### PR DESCRIPTION
**Pull request by SomeshRA**
### JIRA link (if applicable) ###

NFDIV-784

### Change description ###
Integration with Send-Letter Service for NFD Bulk Print

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/divorce/print/BulkPrintServiceTest.java [hunk 1]
```diff
@@ -110,5 +186,154 @@ void shouldReturnLetterIdForValidRequest() {
...
+
+    private User solicitorUser(List<String> solicitorRoles, String userId) {
+        UserDetails userDetails = UserDetails
+            .builder()
+            .roles(solicitorRoles)
+            .id(userId)
+            .build();
+
+        return new User(APP_1_SOL_AUTH_TOKEN, userDetails);
+    }
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/divorce/print/BulkPrintServiceTest.java:213]: private method goes to last after all public methods

File: src/test/java/uk/gov/hmcts/divorce/print/BulkPrintServiceTest.java [hunk 2]
```diff
@@ -110,5 +186,154 @@ void shouldReturnLetterIdForValidRequest() {
...
+
+        assertThatThrownBy(() -> bulkPrintService.print(print))
+            .isInstanceOf(InvalidResourceException.class)
+            .hasMessage("Doc name " + documentUuid);
+    }
+
+    @Test
+    void shouldThrowDocumentDownloadExceptionWhenResponseEntityIsNull() {
+        ListValue<DivorceDocument> divorceDocumentListValue = getDivorceDocumentListValue(() -> null);
+
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/divorce/print/BulkPrintServiceTest.java:250]: extra space ?

Comment (SomeshRA) [reply src/test/java/uk/gov/hmcts/divorce/print/BulkPrintServiceTest.java:239]: Removed extra line

File: src/test/java/uk/gov/hmcts/divorce/print/BulkPrintServiceTest.java (hunk 1 above)
Comment (SomeshRA) [reply src/test/java/uk/gov/hmcts/divorce/print/BulkPrintServiceTest.java]: moved private method to the bottom

**Review by nitinprabhu — APPROVED** [review]

**Pull request merged**